### ISSUE TITLE
Kernel crash on Pixel 7a after update

### ISSUE THREAD

**Issue by attisimon** [GrapheneOS/os-issue-tracker#2774]
Hello! After update the phone restarted, finished the update, I unlocked both my profiles and later I noticed, that the phone has rebooted again. When unlocking it, I was greeted by a kernel crash message, attached.
[kernel_crash_pixel7a.txt](https://github.com/GrapheneOS/os-issue-tracker/files/13596711/kernel_crash_pixel7a.txt)

Also, it looks like the app optimisation did not finish after the first reboot, but only after the second one.

**Comment by thestinger** [comment]
There's a new release coming out today which has a good chance of fixing this. Please report a new issue if it happens again after the next release.

**Comment by thestinger** [comment]
Please reopen if it still happens on the 2023120701 release.
